apis/Google.Cloud.Diagnostics.AspNet/Google.Cloud.Diagnostics.AspNet.Tests/ErrorReporting/GoogleWebApiExceptionLoggerTest.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright 2018 Google Inc. All Rights Reserved.
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Cloud.Diagnostics.Common;
+using Moq;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Web.Http.Controllers;
+using Xunit;
+
+namespace Google.Cloud.Diagnostics.AspNet.Tests
+{
+    public class GoogleWebApiExceptionLoggerTest
+    {
+        private static readonly Exception _exception = new Exception("opps...");
```

Comment:
Super-nit - was this intended to be "oops"?

Reply:
Yes! Fixed here and from where I copied it.